mllib/src/main/scala/org/apache/spark/ml/Pipeline.scala

```diff
@@ -132,7 +133,8 @@ class Pipeline @Since("1.4.0") (
    * @return fitted pipeline
```

Comment:
Where should we document the ml events? Should we add doc at places we add ml events like `Pipeline.fit`? Or write some doc in ml document?

Reply:
Ok. I see.